pkg/apis/autoscaling/v1/conversion.go

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
they're tested by the fuzzer, IIRC, and by a host of other tests, but I can add one if needed.